ara/cli/host.py

```diff
@@ -62,19 +78,17 @@ def get_parser(self, prog_name):
         parser.add_argument(
             'host',
             metavar='<host>',
-            help='Host name or id to show',
+            help='Host id to show',
         )
         return parser
 
     def take_action(self, args):
-        try:
-            host = (models.Host.query
-                    .filter((models.Host.id == args.host) |
-                            (models.Host.name == args.host)).one())
-        except models.NoResultFound:
+        host = models.Host.query.get(args.host)
```

Comment:
The way OSC handles this is that if the search by name returns len > 1, they raise an exception. i.e: https://github.com/openstack/python-openstackclient/blob/master/openstackclient/common/utils.py#L100

We could keep that feature and handle it similarly for fields that are possibly non-unique